src/lightning/app/CHANGELOG.md

```diff
@@ -20,35 +20,29 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 - Renamed `lightning.app.components.LiteMultiNode` to `lightning.app.components.FabricMultiNode` ([#16505](https://github.com/Lightning-AI/lightning/pull/16505))
 
-
 ### Deprecated
 
 -
 
-
 ### Removed
 
 -
 
-
 ### Fixed
 
 - Fixed the Drive root_folder not parsed properly ([#16454](https://github.com/Lightning-AI/lightning/pull/16454))
 
-
```

Comment:
Normally we have 2 newlines between versions

Reply:
artifact from pre-commit, will fix